src/main/java/hu/flowacademy/zetaabsencemanager/service/AdminUserService.java

```diff
@@ -89,16 +89,17 @@ public User updateUser(@NotNull Long id, @NotNull User user) {
     modifyUser.setUpdatedAt(LocalDateTime.now());
     modifyUser.setUpdatedBy(authenticationService.getCurrentUser());
     userRepository.save(modifyUser);
+    userRepository.save(modifyUser);
```

Comment:
Ez miért? A felette lévő sorban lemented

Reply:
késő volt már és benne felejtettem, de végül is nem is kellett hozzányúlnom az updateUser() függvényhez, mert egyszerűbb megoldást választottam. Miután beértem a Flow-ba, javítom.